backend/globaleaks/handlers/whistleblower/submission.py

```diff
@@ -160,10 +166,35 @@ def db_create_receivertip(session, receiver, internaltip, tip_key):
     session.add(receivertip)
     return receivertip
 
+def is_valid_uuid(value):
+    try:
+        return str(uuid.UUID(value, version=4)) == value
+    except ValueError:
+        return False
+
+def find_statistical_info(session, tid:str, answers:dict, default_language:str):
```

Comment:
@alessiofranceschini @joeman65: this function should probably be recursive or it will ignore statistical questions that are part of a group or subgroup.